ui/packages/consul-ui/app/components/toolbar/index.hbs

```diff
@@ -0,0 +1,7 @@
+<nav class="toolbar">
+  <nav class="toolbar-actions">
+    <div class="toolbar-scroller">
+      {{yield}}
+    </div>
+  </nav>
+</nav>
```

Comment:
afaik `nav` should only be used for site / app main nav (or sub nav or whatever) but its generally a way to navigate around the site / app rather than for a toolbar thing. I'd just stick with `div` here unless you can suggest something anything else? (I'd be concerned about suggesting `menu` as I vaguely remember browser difference/complications from a while back but I've no idea if thats still relevant, so unless you know more I'd maybe err on the side of caution with a `div`)

Reply:
I stole this from Vault and I played ignorantly to be honest but yeah, it is especially weird because we have nested nav too. I'll change that.